### ISSUE TITLE
using ASCII.CR and ASCII.LF with put_line ?

### ISSUE THREAD

**Issue by CoghettoR** [zertovitch/ada-pdf-writer#2]
Hi,

Is it possible to have a line feed with _ascii.lf_ or _ascii.cr_ in the put_line procedure, for example ?

```ada
with PDF_Out;

procedure test2 is
   use PDF_Out;
   Doc : PDF_Out_File;
begin
   Doc.Create ("my_pdf.pdf");
   Doc.Page_Setup (A4_portrait);
   Doc.put_line("Hello"&ascii.lf&"Bye.");
   Doc.Close;
end test2;
```

**Comment by zertovitch** [comment]
Not yet, but it is very easy to implement.
We just have to agree what to do with each of LF and CR.
I'd suggest:
* LF: like New_Line
* CR: do nothing
So if there is an streamed import from Linux or Windows texts (LF, or CR & LF) it will work for both.
Does it sound good?

**Comment by zertovitch** [comment]
Implemented in commit 0c4bfe8.
